understanding/22/target-size-minimum.html

```diff
@@ -39,14 +39,14 @@ <h2>Intent of Target Size (Minimum)</h2>
 
-        <p>The requirement is for targets to have at least 24 px offset between them. The offset between targets is the size plus spacing of a target, measured from the nearest point of each adjacent target to the farthest point of the current target. For example, a target of 20 x 20px would meet the requirement if it had a spacing of 2 px on all sides. If the target is 24 x 24px or larger, no spacing between it and adjacent targets would be required.</p>
+        <p>The requirement is for targets to have at least 24 px offset between them. The offset between targets is the size plus spacing between targets, measured from the nearest point of each adjacent target to the farthest point of the current target. For example, a target of 20 x 20px would meet the requirement if it had a spacing of 4 px to any adjacent target. If the target is 24 x 24px or larger, no spacing between it and adjacent targets would be required.</p>
       
 
         <p>There are three exceptions:</p>
         <ol>
-            <li>If the target is inline within a sentence there doesn’t need to be a minimum target size.</li>
-            <li>If one target is entirely enclosed within another target, each target needs to provide a unique target area of at least 24 by 24 CSS pixels.</li>
-            <li>If the spacing of the targets is essential to the information being conveyed there doesn’t need to be a minimum target size.</li>
+            <li>If a target is smaller than 24 x 24px but has sufficient spacing, it passes. The spacing is sufficient if the offset between the farthest edge of the target to the nearest edge of any neighbouring target is 24px or greater. What edge of the target is farthest depends on the position of the neighbouring target to which the offset is established. For example, if the neighbouring target is above it, the farthest edge would be the bottom of the target, and the offset would be measured form this bottom edge to the bottom edge of the neighbouring target above.
+            <lI>The Success Criterion does not apply to inline targets in blocks of text, for example, text links. When multiple links are embedded in blocks of texts in smaller text sizes, maintaining a 24px offset between links in adjacent lines of text would require a large line height which can be undesirable in many design contexts. Also, inline links such as footnoote numbers common in scientific texts may sometimes have a width of well below 24px. Placed in a sequence, these links often have a horizontal offset of less than 24px. Stacked lists of links, as in navigation structures, do not count as inline links.</li>
```

Comment:
"in blocks of text, for example, text links." maybe cleaner to change to "in blocks of text – for example, text links inside a sentence or paragraph."